### PR TITLE
RK-17813 - fix: remove unused memorydb flag

### PR DESCRIPTION
**Pull request by gosharo**

### CODE REVIEW

**Comment by alexeygutkin** [comment]
https://github.com/Rookout/helm-charts/blob/master/charts/rookout-hybrid/templates/datastore-deployment.yaml#L76 why you are not removing it from here?

**Comment by gosharo** [comment]
> https://github.com/Rookout/helm-charts/blob/master/charts/rookout-hybrid/templates/datastore-deployment.yaml#L76 why you are not removing it from here?

nope.

**Comment by Urook** [comment]
> > https://github.com/Rookout/helm-charts/blob/master/charts/rookout-hybrid/templates/datastore-deployment.yaml#L76 why you are not removing it from here?
> 
> removed, it's true by default

@gosharo I think he means you should delete it from:
[The README](https://github.com/Rookout/helm-charts/blob/32bf4245616b2e8e277433368e73f7fd2e9be0f3/charts/rookout-hybrid/README.md?plain=1#L109)

File: charts/rookout-hybrid/README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Rookout Hybrid Architecture
 
-![Version: 0.1.0](https://img.shields.io/badge/Version-0.1.0-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square)
+![Version: 0.3.1](https://img.shields.io/badge/Version-0.1.0-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square)
```
Comment (Urook) [inline charts/rookout-hybrid/README.md:3]: Why is the url still Version-0.1.p-informational?

File: charts/rookout-hybrid/README.md [hunk 2]
```diff
@@ -63,7 +63,7 @@ helm upgrade --install rookout rookout/rookout-hybrid \
     --set rookout.token=<ROOKOUT_TOKEN> \
     --set datastore.enabled=false \
     -n rookout --create-namespace
-```
```
Comment (Urook) [inline charts/rookout-hybrid/README.md:66]: Why did you delete this?

**Review by Urook — CHANGES_REQUESTED** [review]

**Review by alexeygutkin — DISMISSED** [review]

File: charts/rookout-hybrid/README.md (hunk 2 above)
Comment (gosharo) [reply charts/rookout-hybrid/README.md:66]: not intended

**Review by Urook — CHANGES_REQUESTED** [review]

**Review by Urook — APPROVED** [review]

**Pull request merged**